cmd/containerd/command/main.go

```diff
@@ -116,8 +116,12 @@ func App() *cli.App {
 			return errors.Wrap(err, "creating temp mount location")
 		}
 		// unmount all temp mounts on boot for the server
-		if err := mount.CleanupTempMounts(0); err != nil {
-			return errors.Wrap(err, "unmounting temp mounts")
+		warnings, err := mount.CleanupTempMounts(0)
+		if err != nil {
+			log.G(ctx).WithError(err).Error("unmounting temp mounts")
```

Comment:
Do we still want `err` to return here?Looks like the only case where we would get an error is if `getmntinfo` returns an error

Reply:
ya, i think this is fine